### ISSUE TITLE
Reopen #7: pLDDT not writing to PDB

### ISSUE THREAD

**Issue by LandonGetz** [evolutionaryscale/esm#191]
I am having the same issue as was previously reported in #7, where all pLDDT values are being stored as 1.00. Including print(protein.plddt) provides the per-residue tensor values for pLDDT (values less than 1), so I know the model is generating them. I am running code that is very similar to #7.

Am I missing something obvious or has that previous bug popped back up?

**Comment by LandonGetz** [comment]
I think this issue is in handling of the protein_complex into .to_pdb. If I print the output of 

'protein_complex = protein.to_protein_complex()
for chain in protein_complex.chain_iter():
    print(chain.confidence)'

I get an array of 1's (which is what is being output into the b-factor column of my PDBs). I've managed to hack my way out of this by running protein.to_pdb_string() and then writing the PDB file manually:

`scaled_plddt = protein.plddt * 100  # Scale pLDDT to 0-100
 protein_chain = protein.to_protein_chain()
 protein_chain.confidence = scaled_plddt.detach().cpu().numpy()
 pdb_string = protein_chain.to_pdb_string()
 with open(out_pdb, "w") as f:
        f.write(pdb_string)`

Let me know if I'm just missing something silly, or if this is a bona fide bug

**Comment by ebetica** [comment]
cc @imathur1 , I think this is a real bug?
